src/ApiException.cs

```diff
@@ -4,6 +4,9 @@
 
 namespace Laserfiche.Api.Client
 {
+    /// <summary>
+    /// An exception thrown from error API responses.
```

Comment:
It reads to me like an exception is thrown from a response, while the exception is just included in the response. But I can't make a good suggestion

Reply:
I followed a similar format to dotnet Exception class description `Represents errors that occur during application execution.`
https://learn.microsoft.com/en-us/dotnet/api/system.exception?view=net-7.0